endpoints/api/namespacequota.py

```diff
@@ -31,12 +30,18 @@
 logger = logging.getLogger(__name__)
 
 
-def quota_view(quota):
+def quota_view(quota, default_config=False):
```

Comment:
you dont do anything with default_config accept set it, recommend not being a parameter

Reply:
I am using it [here](https://github.com/quay/quay/pull/1280/files#diff-97350d41b191619358d2e04565d8f58061faafcf99224960fe1cd38bc9d79846R76). To show the note basically.